README.md

```diff
@@ -424,20 +424,20 @@ See the ["Flow Chart"](#flow-chart) section for a summary of the asymmetry in th
   3. *Server* returns a shuffled list of `commons.MAX_MESSAGES` (*i+j*) tuples of *(<sup>[0-i]</sup>pmgdh,<sup>[0-i]</sup>enc_mid) U (<sup>[0-j]</sup>decoy_pmgdh,<sup>[0-j]</sup>enc_mid)*
 
 
-### Journalist message id fetching protocol
+### Source message id fetching protocol
   1. *Source* derives *SC<sub>SK</sub> = G(KDF(fetching_salt + PW))*
   2. *Source* fetches *(<sup>[0-n]</sup>pmgdh,<sup>[0-n]</sup>enc_mid)* from *Server* (`n=commons.MAX_MESSAGES`)
-  3. For every *(<sup>n</sup>pmgdh,<sup>n</sup>enc_mid)*:
-     - *Source* calculates *<sup>n</sup>kmid = DH(<sup>n</sup>pmgdh,SC<sub>SK</sub>)*
-     - *Source* attempts to decrypt *<sup>n</sup>mid = Dec(<sup>n</sup>kmid,<sup>n</sup>enc_mid)*
-     - If decryption succeeds, save *<sup>n</sup>mid*
+  3. For every *(<sup>i</sup>pmgdh,<sup>i</sup>enc_mid)*:
```

Comment:
Do you think we should conventionally always use `i`, `j`, `k` as indexes throughout the readme and the flow diagram? Asking because there are a lot of references to `n` overall and I am wondering if I should go over all of them

https://github.com/freedomofpress/securedrop-protocol/blob/d86b80c31df8b04bd6ff4f0038f769cba41002a4/README.md?plain=1#L361

https://github.com/freedomofpress/securedrop-protocol/blob/d86b80c31df8b04bd6ff4f0038f769cba41002a4/README.md?plain=1#L365

https://github.com/freedomofpress/securedrop-protocol/blob/d86b80c31df8b04bd6ff4f0038f769cba41002a4/README.md?plain=1#L610

If so, also the `n=commons.MAX_MESSAGES` should change?

Reply:
Yes, I recommend using $i,j,k$ as "stateless" indices. Using $n$ as the max. number of messages is fine tough. For the number of actually sent messages I would go for $m$. Alternatively you can use notations such as $\texttt{MAX\\_MESSAGES}$ and $\texttt{N\\_SENT\\_MESSAGES}$.